src/app/form/page.tsx

```diff
@@ -36,7 +36,7 @@ export default function Form() {
   };
 
   const sendEventTracking = (index: number) => {
-    window.parent.postMessage('multistep_step_' + index, 'https://holaglow.com');
+    window.parent.postMessage('(googleevent)multistep_step_' + index, 'https://holaglow.com');
```

Comment:
```suggestion
    window.parent.postMessage('(googleevent)multistep_step_' + index + '|', 'https://holaglow.com');
```

Hago un split por |, lo primero es el tipo y lo segundo datos extra si se quieren enviar

Reply:
@rubenausinholaglow  entiendo que es necesario el separador igualmente aunque no haya info extra (?)

Reply:
Yes!